docs/content/en/concepts/quickstart-tenancy-and-apis.md

```diff
@@ -309,19 +369,14 @@ some way of securely distinguishing them.
 Each `APIExport` is allocated a randomized private secret - this is currently just a large random number - and a public
 identity - just a SHA256 hash of the private secret - which securely identifies this `APIExport` from others.
 
-This is important because an `APIExport` makes a virtual workspace available to interact with all instances of a
-particular `APIResourceShema`, and we want to make sure that users are clear on which service provider `APIExports` they
-are trusting and only the owners of those `APIExport` have access to their resources via virtual workspaces.
+This is important because an `APIExport` makes service endpoints available to interact with all instances of a particular `APIResourceShema`, and we want to make sure that users are clear on which service provider `APIExports` they are trusting and only the owners of those `APIExport` have access to their resources via the service endpoints.
 
-Q: Why do you have to use `--all-namespaces` with the apiexport virtual workspace?
+Q: Why do you have to use `--all-namespaces` with the `APIExport` service endpoint?
 
-A: Think of this virtual workspace as representing a wildcard listing across all workspaces. It doesn't make sense to
-look at a specific namespace across all workspaces, so you have to list across all namespaces too.
+A: Think of this endpoint as representing a wildcard listing across all workspaces. It doesn't make sense to look at a specific namespace across all workspaces, so you have to list across all namespaces too.
```

Comment:
Out of curiosity, why doesn't it make sense to only look at a single namespace (or a set of) across many workspaces?

Reply:
This is not from me so that I can only guess. One reason may be that the natural isolation level in kcp is the workspace rather than namespace.
I only intended to remove "virtual workspace" as we want to move away from this wording that has caused confusion.